bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Voice.java

```diff
@@ -157,14 +158,14 @@ public static String interpret(@ParamDoc(name = "text") Object text) {
      * In case of interpretation error, the error message is played using the default audio sink.
      *
      * @param text The text to interpret
-     * @param interpreter The Human Language Interpreter to be used
+     * @param interpreters Comma separated list of human language text interpreters to use
      */
     @ActionDoc(text = "interprets a given text by a given human language interpreter", returns = "human language response")
```

Comment:
"interprets a given text by given human language interpreter(s)"

Reply:
Please do not break API. Add new action which accepts a `List<String>`. You can call the new action from the "old" one by using `List.of(interpreter)`. Also below at other places.

Reply:
This is not a breaking change because the parameter is a string.
But why not adding a new method for the case we want to provide a list of interpreters.
But I am not sure that inside a DSL rule for example, it is possible to build a List of objects.